.travis.yml

```diff
@@ -47,6 +47,35 @@ matrix:
            PROJECT_URL=https://github.com/joblib/joblib.git
       python: 3.7
       if: commit_message =~ /(\[ci downstream\]|\[ci joblib\])/
+    - os: linux
+      env: PROJECT=ray TEST_REQUIREMENTS="setproctitle hypothesis psutil"
+           PROJECT_URL=https://github.com/ray-project/ray.git
+           PYTEST_ARGS="--duration=10 -k test_mini or test_basic"
+      python: 3.7
+      if: commit_message =~ /(\[ci downstream\]|\[ci ray\])/
+      install:
+        - pip install .
+        - pip install --upgrade -r dev-requirements.txt
+        - pip install tornado
+        - if [[ $TRAVIS_PYTHON_VERSION != 'pypy'* ]]; then
```

Comment:
I don't think this if statement is necessary.

Reply:
We also don't need `scipy`

Reply:
see above comment for `tornado`